dll/win32/shell32/CExtractIcon.cpp

```diff
@@ -138,6 +138,21 @@ HRESULT STDMETHODCALLTYPE CExtractIcon::SetNormalIcon(
 {
     TRACE("(%p, %s, %d)\n", this, debugstr_w(pszFile), iIcon);
 
+    if (lstrcmpiW(PathFindExtensionW(pszFile), L".exe") == 0)
```

Comment:
Not .com files too?

Reply:
![com-has](https://user-images.githubusercontent.com/2107452/69423220-e0f2f480-0d68-11ea-89b4-702775c2f6d8.png)
`*.com` has the default icon and there is no file specific icon.

```txt
HKCR,"comfile","",0x00000000,"DOS Application"
...
HKCR,"comfile\DefaultIcon","",0x00020000,"%SystemRoot%\system32\shell32.dll,-3"
```